app/src/main/java/ar/com/wolox/android/example/network/UserService.java

```diff
@@ -0,0 +1,21 @@
+package ar.com.wolox.android.example.network;
+
+import java.util.List;
+
+import ar.com.wolox.android.example.model.User;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+/**
+ * User service - Interface used with retrofit
+ */
+public interface UserService {
+    String USERS_COLLECTION = "/users";
+    String EMAIL = "email";
+    String PASSWORD = "password";
+
+    @GET(USERS_COLLECTION)
```

Comment:
In this case you can write the string hardcode. It´s just more easy to follow when you have endless enpoints xD